src/_utils/item/index.tsx

```diff
@@ -79,5 +79,5 @@ const StyledItem = styled(Item)`
   }
 `
 
-export { ItemStatus } from './Item'
+export { ItemStatus, ItemProps } from './Item'
```

Comment:
Unless you intent to export them outside of the lib, don't expose the props in this file. Also Typescript raises a warning why trying to export interfaces this way, dunno why =/